src/components/UnsupportedMicrobitWarningDialog.svelte

```diff
@@ -18,24 +19,12 @@
     target: '_blank',
     rel: 'noopener',
   });
-  const linkToBluetoothFlowId = 'link-to-bluetooth';
-  const linkToBluetoothFlow = linkWithProps(
-    {
-      id: linkToBluetoothFlowId,
-    },
-    { tag: 'button' },
-  );
-  // The elements in translated strings don't support Svelte event handlers
-  // so we wire it up ourselves.
-  const wireLinkToBluetoothFlow = (node: Element) => {
-    const link = node.querySelector('#' + linkToBluetoothFlowId);
-    link?.addEventListener('click', onStartBluetoothClick);
-    return {
-      destroy() {
-        link?.removeEventListener('click', onStartBluetoothClick);
-      },
-    };
-  };
+  const linkWithPropsForMicrobitBluetoothSupport = linkWithProps({
+    href: 'https://support.microbit.org/support/home',
```

Comment:
TODO: update link.

Reply:
TODO added in code [b29a676](https://github.com/microbit-foundation/ml-trainer/pull/245/commits/b29a676557add0e4f30a3394e1f59f38b5bf9a99)